frontend/app/globals.css

```
@@ -1,21 +1,40 @@
+  --surface: rgb(229, 231, 224);
+  --border: rgb(205, 208, 198);
+  --muted: #7c7f74;
+  --accent: #1d1b16;
+  --accent-text: rgb(243, 244, 240);
+  --table-row-height: 34px;
+  --table-header-height: 32px;
+  --table-cell-px: 12px;
+  --table-cell-py: 8px;
 }
 
 @theme inline {
```

Stylelint (17.11.1)

[error] Unexpected unknown at-rule "`@theme`" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)